test/bids_examples_eeg_rest_fmri.jsonld

```diff
@@ -507,29 +507,6 @@
     {
       "@id": "_:000035",
       "@type": "https://openminds.ebrains.eu/core/File",
-      "IRI": "PREFIXacq-NODDI66DIR_dwi.json",
```

Comment:
what happened to these files?

Reply:
This file got deleted:  `PREFIXacq-NODDI66DIR_dwi.json` in this [commit](https://github.com/bids-standard/bids-examples/commit/bf6144c7069cc795aa8ed58061e0cd12ae18d8d0) to bids example 
As it was one of the first one that got detected, so after removing it @id of all the files after that one got deleted.